src/hacbs/hacbsFeatureFlag.ts

```diff
@@ -1,8 +1,13 @@
 import { SetFeatureFlag } from '@openshift/dynamic-plugin-sdk';
 
 export const HACBS_FLAG = 'HACBS';
+export const IS_HACBS_DEFAULT = 'HACBS';
 
 export const enableHACBSFlagFromQueryParam = (setFlag: SetFeatureFlag): void => {
+  if (IS_HACBS_DEFAULT) {
+    setFlag(HACBS_FLAG, true);
+    return;
+  }
   let enabled = false;
```

Comment:
We just need to set the default value but still allow for overriding via url and local storage.

```suggestion

export const enableHACBSFlagFromQueryParam = (setFlag: SetFeatureFlag): void => {
  let enabled = true;
```

Further down we must also change how we read from localStorage:
```
    enabled = localStorage.getItem('hacbs') === 'true';
```
to:
```
    enabled = localStorage.getItem('hacbs') !== 'false';
```

Reply:
updated